app/models/encryptable/ose_secret.rb

```diff
@@ -2,7 +2,7 @@
 
 require_relative '../../utils/crypto/symmetric/aes256iv'
 
-class Encryptable::OSESecret < Encryptable
+class Encryptable::OseSecret < Encryptable
```

Comment:
if we rename this, we need to create a db migration to change given values in encryptables#type column.

could u please create a migration for this?

Reply:
similar to: https://github.com/puzzle/cryptopus/blob/master/db/migrate/20220113085536_rename_accounts_to_encryptables.rb#L13